cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -174,6 +174,7 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
         try {
             repository.delete(releaseName_build)
         } catch (_: Exception) {
+            // Intentional: Ignore exception as it occurs frequently when the repository does not exist, no critical impact
```

The provided fix strategy requires us to simply add a comment within the catch block to explain why it is empty. The expert information indicates that the catch block is there to handle exceptions that occur when trying to delete a repository that may not exist: such exceptions are non-critical for the operation of the surrounding code. Therefore, an appropriate comment explains that ignoring the exception is intentional, as the situation (repository not existing) is a known and non-problematic state.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)